packages/orbit-tailwind-preset/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@kiwicom/orbit-tailwind-preset",
+  "version": "1.0.0",
+  "description": "Orbit tailwind preset which is based on orbit-design-tokens",
+  "author": "Kiwi.com",
+  "license": "MIT",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kiwicom/orbit",
+    "directory": "packages/orbit-tailwind-preset"
+  },
+  "files": [
+    "README.md",
+    "dist/*"
+  ],
+  "keywords": [
+    "orbit",
+    "tailwind",
+    "preset",
+    "design-system",
+    "tokens"
+  ],
+  "scripts": {
+    "build": "yarn tsup",
+    "pretest": "yarn workspace @kiwicom/orbit-design-tokens build && yarn build && yarn tailwindcss -o style.css",
```

Comment:
Unfortunately, this only works if the test command is executed directly on the package. I am used to running `yarn test` from the root folder, sometimes with the `--watch` flag. And if for some reason I don't have the `style.css` generated (or updated) it will fail because it can't access the file. I understand this only happens when one uses it for the first time, but is it ideal? As the styles.css is untracked I don't know if we should rely on it to run tests. I see that for the pipeline this is being built before executing the tests, but that does not happen locally, and one can develop and work on components without ever generating the `style.css`, that is why I would like to avoid the need to manually execute this script before the tests.
If one clones the repository for the first time, builds everything and runs tests… they fail. They won't know that this `pretest` needs to be executed. What would be the problem of having the `yarn tailwindcss -o style.css` as part of the `build` script? 🤔 

This is non-blocking, but I think the experience could be optimized here. WDYT?

Reply:
the problem is that it's been running with jest from the root folder, while it should be run with Lerna for all workspaces to execute test commands in each workspace. This problem will not happen with nx, but I can add it here with Lerna. pre commands executed automatically if a test command is run but just does not do it from the test command in the root. :)